packages/@aws-cdk/aws-dynamodb/lib/table.ts

```diff
@@ -661,39 +765,175 @@ abstract class TableBase extends Resource implements ITable {
   /**
    * Metric for the system errors this table
    *
-   * @default sum over a minute
+   * @deprecated use `metricSystemErrorsForOperations`.
    */
   public metricSystemErrors(props?: cloudwatch.MetricOptions): cloudwatch.Metric {
+
+    if (!props?.dimensions?.Operation || !props?.dimensions?.TableName) {
```

Comment:
:(.

This doesn't jibe with all other metrics which add in `TableName` automatically (which is like half of the point of the CDK). Can you not add in TableName automatically like before and just expect users to pass `Operation` ?

Reply:
(I know it makes the dimension override behavior a bit wonky... but not TOO much)

Reply:
I considered it, but honestly since this method is deprecated, i think its ok to not invest effort in it.
Users can and should use `table.metricSystemErrorsForOperations({ operations: [GET_ITEM]})` - which doesn't require the table name.

Reply:
Supported now. Users don't need to pass in the TableName